network/src/commonMain/kotlin/com/wire/kalium/network/api/auth/AuthApiImp.kt

```diff
@@ -10,10 +10,10 @@ import io.ktor.client.statement.HttpResponse
 import io.ktor.http.Cookie
 
 class AuthApiImp(private val httpClient: HttpClient) : AuthApi {
-    override suspend fun renewAccessToken(cookie: Cookie): KaliumHttpResult<RenewAccessTokenResponse> =
+    override suspend fun renewAccessToken(cookieName: String, cookieValue: String): KaliumHttpResult<RenewAccessTokenResponse> =
```

Comment:
```suggestion
    override suspend fun renewAccessToken(refreshToken: String): KaliumHttpResult<RenewAccessTokenResponse> =
```

Reply:
I guess we can consider `zuid` as something static, written in stone, just like it has been for the other projects before us.

I'll remove the `cookieName` and assume `zuid` it is.